### PR TITLE
Angi devel to be merged into main

### PR DESCRIPTION
**Pull request by fmherschel**
Upgrade to 1.2.6

### CODE REVIEW

File: man-tester/SAPHanaSR-testCluster.8 [hunk 1]
```diff
@@ -173,7 +174,7 @@ F.Herschel, L.Pinne.
 .PP
 .\"
 .SH COPYRIGHT
-(c) 2023 SUSE LLC
+(c) 2023-2014 SUSE LLC
```
Comment (angelabriel) [inline man-tester/SAPHanaSR-testCluster.8:177]: Typo 2014 --> 2024

File: man/SAPHanaSR_maintenance_examples.7 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .\" Version: 1.001 
 .\"
-.TH SAPHanaSR_maintenance_examples 7 "06 Dec 2023" "" "SAPHanaSR"
+.TH SAPHanaSR_maintenance_examples 7 "25 Jan 2023" "" "SAPHanaSR"
```
Comment (angelabriel) [inline man/SAPHanaSR_maintenance_examples.7:3]: Typo in the year --> 2024

**Review by angelabriel — CHANGES_REQUESTED** [review]
Looks good to me. Only 2 typos in the date of 2 man pages found.  Would be nice if that can be fixed before we merge. 
And the script `tools/SAPHanaSR-upgrade-to-angi-demo` needs some love to pass the **shellcheck** (or simply exclude this file from the check for now)
Thanks

File: man-tester/SAPHanaSR-testCluster.8 (hunk 1 above)
Comment (fmherschel) [reply man-tester/SAPHanaSR-testCluster.8]: DONE

File: man/SAPHanaSR_maintenance_examples.7 (hunk 2 above)
Comment (fmherschel) [reply man/SAPHanaSR_maintenance_examples.7]: DONE

File: man-tester/SAPHanaSR-testCluster.8 (hunk 1 above)
Comment (angelabriel) [reply man-tester/SAPHanaSR-testCluster.8]: Thanks

File: man/SAPHanaSR_maintenance_examples.7 (hunk 2 above)
Comment (angelabriel) [reply man/SAPHanaSR_maintenance_examples.7]: Thanks

**Comment by fmherschel** [comment]
Also shellcheck should be satisfied now.

**Pull request merged**